### PR TITLE
Reset events on month change

### LINKED ISSUE #105: Returning to a month shows events for previously focused day but that day is no longer focused
* Open the calendar widget
* Select a day that has events - that day has a blue circle (focused)
* Move to another month
* Move back to the initial month
* The events still show but there is no focused day (no blue circle).

### PR DESCRIPTION
**Pull request by peteruithoven**
Fixes #105 

By resetting selected_date.

Events are otherwise not cleared when navigating between months.

### CODE REVIEW

**Comment by jeremypw** [comment]
I did not realise this was already in the pipeline when I created the issue!

**Comment by peteruithoven** [comment]
How do I keep forgetting that rule :man_facepalming:
Thanks for the review.

File: src/Indicator.vala [hunk 1]
```diff
@@ -96,10 +96,13 @@ public class DateTime.Indicator : Wingpanel.Indicator {
         if (event_grid != null) {
             event_grid.destroy ();
         }
-        if (calendar.selected_date == null) return false;
+        if (calendar.selected_date == null){
```
Comment (jeremypw) [inline src/Indicator.vala]: Space missing before brace.

Comment (peteruithoven) [reply src/Indicator.vala]: :man_facepalming: thanks, fixed.

**Pull request merged**